pkg/channel/types.go

```diff
@@ -51,30 +51,3 @@ const (
 func (t Type) String() string {
 	return [...]string{"LISTENER", "SENDER", "EVENT", "STATUS"}[t]
 }
-
-// EventDataType types passed to cloud events
-type EventDataType string
-
-const (
-	// PTPEvent event type is ptp
-	PTPEvent EventDataType = "PTP_EVENT"
-	// PTPStatus event type is ptp status
-	PTPStatus = "PTP_STATUS"
-	// HWEvent event type is hw
-	HWEvent = "HW_EVENT"
-	// Event event type is generic event
-	Event = "EVENT"
```

Comment:
what is the risk of defaulting to generic EVENT, right now we don't handle EVENT data Type right ?

Reply:
default EVENT deleted. v1 CloudNativeEvent() will return event with empty string as EventType.